### PR TITLE
Clock frequencies in bionicc.cpp

### PR DESCRIPTION
**Pull request by Asayuki-K**
First pull request ever, never used git before.
I added a few information in the comments, most notably the clock frequency of the C8751H-88 protection MCU which was unknown so far.

### CODE REVIEW

**Comment by cuavas** [comment]
Waiting to see if we can get raw screen timings as well.

**Comment by angelosa** [comment]
The whole SCREEN_RAW_PARAMS system needs some kind of searchable string in it, so I'll just merge for now.

**Comment by cuavas** [comment]
You've got two back porches in that comment :P

File: src/mame/drivers/bionicc.cpp [hunk 1]
```diff
@@ -11,14 +11,37 @@
...
+    hsync:               5.312us,  32 pixels
+    back porch + sync:  15.106us,  91 pixels
+    active video:       42.662us, 257 pixels (it looks like the first pixel is repeated)
+    front porch:         6.308us,  38 pixels
+
+    vtotal:             16.660ms, 260 lines
+    vsync:             256.304us,   4 lines
+    back porch + sync:   1.282ms,  20 lines
+    active video:       14.353ms, 224 lines
+    back porch:          1.025ms,  16 lines
```
Comment (Asayuki-K) [inline src/mame/drivers/bionicc.cpp]: Whoops. Yes, line 36 is supposed to be front porch, not back porch.

**Pull request merged**